packages/addons-manager/bundle/config/upstream/addonconfigscrds/csi.tanzu.vmware.com_azurefilecsiconfigs.yaml

```diff
@@ -60,7 +58,9 @@ spec:
             description: AzureFileCSIConfigStatus defines the observed state of AzureFileCSIConfig
             properties:
               secretRef:
-                description: Name of the secret created by csi controller
+                description: 'INSERT ADDITIONAL STATUS FIELD - define observed state
```

Comment:
don't commit other files generated, you can ignore them and let the owner fix.

Reply:
thanks, I will revert this change